### PR TITLE
docs(flow): wording, formatting

### PR DESCRIPTION
**Pull request by alexcg1**
Just a few fixes for #812 to simplify wording and improve formatting.

@JoanFM Asking you to review, since you wrote it in the first place and you can see if I reworded wrongly :)

### CODE REVIEW

File: docs/chapters/flow/index.md [hunk 1]
```diff
@@ -30,34 +30,35 @@ from jina.flow import Flow
 f = Flow()
 ```
 
-`Flow()` accepts some arguments, see `jina flow --help` or [our documentation](https://docs.jina.ai) for details. For example, `Flow(log_server=True)` will enable the logs emission to the [dashboard](https://github.com/jina-ai/dashboard). 
+`Flow()` accepts some arguments, see `jina flow --help` or [our documentation](https://docs.jina.ai) for details. For example, `Flow(log_server=True)` will send logs to the [dashboard](https://github.com/jina-ai/dashboard).
```
Comment (JoanFM) [inline docs/chapters/flow/index.md:33]: enable seems more accurate?

File: docs/chapters/flow/index.md [hunk 2]
```diff
@@ -296,23 +296,20 @@ jina flow --yaml-path myflow.yml
...
-# Common design patterns
+# Common Design Patterns
 
-jina is a really flexible AI-powered neural search framework. It is designed to enable any pattern that can be framed as  
-a neural search problem.
-However, there are basic common patterns that show up when developing search solutions with jina and here is a recopilation of some of them.
+Jina is a really flexible AI-powered neural search framework and is designed to enable any pattern that can be framed as a neural search problem. However, there are basic common patterns that show up when developing search solutions with Jina:
 
-- CompoundIndexer (Vector + KV Indexers):
+## CompoundIndexer (Vector + KV Indexers)
```
Comment (JoanFM) [inline docs/chapters/flow/index.md:310]: 100% needed! It looked awful before

File: docs/chapters/flow/index.md [hunk 3]
```diff
@@ -332,16 +329,11 @@ metas:
   name: complete indexer
 ```
 
-This type of construction will act as a single `indexer` and will allow to seamlessly `query` this index with the `embedding` vector coming
-from any upstream `encoder` and obtain in the response message of the pod the corresponding `binary` information stored in
-the key-value index. This lets the `VectorIndexer` be responsible to obtain the most relevant documents by finding similarities
-in the `embedding` space while targeting the `key-value` database to extract the meaningful data and fields from those relevant documents.
+This acts as a single indexer, letting you seamlessly query the index with the embedding vector from any upstream encoder. It returns the binary information in the key-value index in the Pod's response message. This lets the `VectorIndexer` be responsible for getting the most relevant Documents by finding similarities in the embedding space while targeting the key-value database to extract meaningful data and fields from the relevant Documents.
```
Comment (JoanFM) [inline docs/chapters/flow/index.md:332]: Totally my fault, but can we think of expressing this with shorter sentences? it is tiring to read right?

File: docs/chapters/flow/index.md (hunk 2 above)
Comment (JoanFM) [reply docs/chapters/flow/index.md:310]: I like the rewording

File: docs/chapters/flow/index.md (hunk 3 above)
Comment (alexcg1) [reply docs/chapters/flow/index.md:332]: Yeah, exhausting ;) I'll break it down more

File: docs/chapters/flow/index.md (hunk 1 above)
Comment (alexcg1) [reply docs/chapters/flow/index.md:33]: Re-adding `enable`

**Comment by alexcg1** [comment]
I've used bullet points to break down a lot of long sentences. You could say I've `Chunk`ed the `Document` ;)

**Pull request merged**